azure-gaming-pr/reference-architectures/unreal-pixel-streaming-in-azure.md

```diff
@@ -13,15 +13,19 @@ ms.prod: azure-gaming
 
 ### Quick-Start Guide
 
 Follow this guide to learn the basics of deploying Pixel Streaming on an Azure GPU Virtual Machine or multiple VMs if using the Matchmaker service to load balance across many streams on multiple GPU VMs: [Quick-Start Guide](unreal-pixel-streaming-deploying.md).
 
 ### Autoscaling Solution
 
-For deploying Unreal Pixel Streaming at scale (i.e., 2 to 1000&#39;s of streams) with autoscaling, multiple streams per GPU and Azure metrics, refer to the Azure documentation here: [Unreal Pixel Streaming at Scale](unreal-pixel-streaming-at-scale.md).
+For deploying Unreal Pixel Streaming at scale (i.e., 2 to 1000&#39;s of streams) with autoscaling, multiple streams per GPU and Azure metrics, refer to the Azure documentation here: [Unreal Pixel Streaming at Scale](unreal-pixel-streaming-at-scale.md). This is an older version, and the newest version is found in the Azure Marketplace.
+
+### Azure Marketplace Solution
+
+Officially published by Epic Games in the Azure Marketplace, this easy to deploy solution builds on top of the Autoscaling Solution and adds even more functionality like lifecyle management, a custom dashboard for managing the deployments, streaming metrics, and many more improvements. All new improvements are done in the Marketplace solution, and the GitHub (i.e., Terraform) will not be updated other than fixing issues. Please refer to the Azure Marketplace solution documentation here: [Unreal Pixel Streaming at Scale](https://docs.unrealengine.com/4.27/ProductionPipelines/CloudDeployments/AzurePixelStreaming/).
```

Comment:
As described before, don't use "i.e."

"Please refer to the Azure Marketplace solution documentation here: [Unreal Pixel Streaming at Scale](https://docs.unrealengine.com/4.27/ProductionPipelines/CloudDeployments/AzurePixelStreaming/)." ->

Suggestion:
Drop "please"
"Documentation for this solution is available as part of Unreal Engine's cloud deployment documentation&mdash;[Unreal Pixel Streaming at Scale](https://docs.unrealengine.com/4.27/ProductionPipelines/CloudDeployments/AzurePixelStreaming/)."